components/app/landing/LandingLatestProjects.vue

```diff
@@ -5,7 +5,7 @@
     <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 2xl:grid-cols-5 gap-4 mt-10">
       <div v-for="project in projects" :key="project.title">
       <NuxtLink to="/project-overview">
-      <div class="shadow p-4 pb-12 text-center">
+      <div class="shadow p-4 pb-12 h-275px text-center">
```

Comment:
Question (blocking): Do you need to write it as `h-275px` does `h-275` work here as well? What's the best practice?